### PR TITLE
Added the ssh port to the SSH User Check trigger

### PR DESCRIPTION
**Pull request by vviveksharma**
### :nut_and_bolt: Description: What code changed, and why?
Added the SSH port functionality to SSH User Trigger API

### :+1: Definition of Done
- The batch check api should be able to run ssh user check to all the nodes with ssh_port as the input in the request body
### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
 - The response when calling batch check to check the SSH User Access 
 

Screenshot 2023-06-13 at 8 23 57 PM

- Demo Video explaining the fixes made 
 https://progresssoftware.sharepoint.com/:v:/s/ChefCoreC/ER2gDAJbiQ9Psc8oBzNJIoUBnEE8Adm9zYhkBOu5oL7anA?e=7xQQe7

### CODE REVIEW

**Comment by Sunanda-Boorla** [comment]
In verify_private pipeline, `chef-server` and `backup to gcs` is failing. Please check that.

File: components/automate-cli/pkg/verifyserver/server/api/v1/sshusercheck.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 
 func (h *Handler) CheckSshUser(c *fiber.Ctx) error {
 	req := new(models.SshUserChecksRequest)
+	c.Request().Header.Set("Content-Type", "application/json")
```
Comment (vipin230) [inline components/automate-cli/pkg/verifyserver/server/api/v1/sshusercheck.go:11]: move this to client from where api request is trigger

Comment (vviveksharma) [reply components/automate-cli/pkg/verifyserver/server/api/v1/sshusercheck.go]: Done changed its location to trigger.go

**Review by vipin230 — APPROVED** [review]

**Review by bvtejaswi — APPROVED** [review]

**Review by rishabhjhs — APPROVED** [review]

**Review by vivekshankar1 — APPROVED** [review]

**Review by punitmundra — APPROVED** [review]
Resolve the comments, then we will merge this.

**Review by Sunanda-Boorla — APPROVED** [review]

**Review by prasad927 — APPROVED** [review]

**Pull request merged**